testing/trino-benchto-benchmarks/src/test/resources/hive_metadata/README.md

```diff
@@ -0,0 +1,7 @@
+These metadata files were generated from the glue connector from TPCDS/TPCH schema with sf1000.
+So, there are no min/max statistics for char-based columns.
+
+### How to generate?
+1. Execute `gimme-aws-creds --mfa-code <code>` in your cmdline to fetch
+   aws credentials.
+2. Run `HiveMetadataRecorder.UpdateMetadataFiles.main()`
```

Comment:
this is for partitioned/unpartitioned data?

Reply:
yes, it's for both